ord_schema/validations.py

```diff
@@ -446,6 +447,22 @@ def validate_reaction(message: reaction_pb2.Reaction,
 
 def validate_reaction_identifier(message: reaction_pb2.ReactionIdentifier):
     check_type_and_details(message)
+    if Chem and message.type in [
+            message.REACTION_SMILES, message.REACTION_CXSMILES
+    ]:
+        if message.type == message.REACTION_CXSMILES:
+            smiles = message.value.split()[0]
+        else:
+            smiles = message.value
+        try:
+            rxn = rdChemReactions.ReactionFromSmarts(smiles, useSmiles=True)
+            if rxn is None:
+                warnings.warn(
+                    f'RDKit {RDKIT_VERSION} could not validate'
+                    f' REACTION_{{CX}}SMILES identifier {message.value}',
+                    ValidationError)
```

Comment:
For consistency with `validate_compound`, which uses `message_helpers.check_compound_identifiers`, let's make this a ValidationWarning not a ValidationError

Reply:
done

Reply:
Actually, we use warning for NAME and error for SMILES (in validate_compound_identifier). The check_compound_identifiers check is less strict, but that's more designed for checking consistency than parse-ability...WDYT?

Reply:
(Just to be clear, the current code will issue a ValidationError if the SMILES can't be parsed for a compound.)

Reply:
Oh, sorry you're right -- I was going off of name and forgot that was only for a consistency check. In that case, let's stick to ValidationError as you have it